c_api/concurrentqueue.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifdef _WIN32
+#if defined(MOODYCAMEL_STATIC) //preferred way
+#define MOODYCAMEL_EXPORT
+#elif defined(DLL_EXPORT)
+#define MOODYCAMEL_EXPORT __declspec(dllexport)
+#else
+#define MOODYCAMEL_EXPORT __declspec(dllimport)
+#endif
+#else
+	
+#error *nix support is untested
+/*
+need non-windows version here
+*/
+#endif
+
+#define MOODYCAMEL_OK 0
```

Comment:
These names risk running into conflicts with the C-API constants for the readerwriterqueue. Consider simplifying the API to return just true/false with no error codes (in C, simply crashing when passed a null pointer is generally acceptable, especially in high-performance code).

Reply:
ok but I don't think C has bool

Reply:
modern C has bool with `<stdbool.h>` but I agree with 1/0 being sufficient.